src/renderer/components/LoadingScreen/index.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export {default} from './LoadingScreen';
```

Comment:
Nit: do we need this connector for anything? Otherwise can't we just import the component directly?

Reply:
0/5 really, there was a conversation a while back about using index files in component folders on the webapp side to explicitly create a public api for said components.

Since the other component in that folder is a child component, it maybe makes it a bit easier to see that it's 'private'? Again, 0/5.